script.js

```diff
@@ -330,6 +382,22 @@ function play() {
   window.speechSynthesis.resume();
 }
 
+async function playArticle(title) {
+  try {
+    const article = await getContent(title)
+    if (state.playing) {
+      state.interjectArticle = article
+      forward()
+
+    } else {
+      next(article)
+    }
+
+  } catch(e) {
+    console.error(e)
+  }
+}
+
```

Comment:
selected article is requested then either interjected into the current trip article reading loop or start the reading loop with the article